### PR TITLE
[UI/UX:InstructorUI] Upload Student Photos Instructions

### PR DESCRIPTION
**Pull request by musaabimran**
### What is the new behavior?
Minor refactor for uploading student photos, as it some times ignored by the user.
**Before:**
image

**After:**
image

### CODE REVIEW

**Comment by williamjallen** [comment]
Can you please add before/after screenshots to your PR description?  Right now, it's unclear whether the screenshot you included is the new or old behavior.

File: site/app/templates/grading/UploadImagesForm.twig [hunk 1]
```diff
@@ -4,9 +4,9 @@
 {% block body %}
     <div class="flex-col flex-col-space">
         <p>
-            Instructions to download images of students:
+            <strong>Instructions to upload images of students:</strong>
```
Comment (williamjallen) [inline site/app/templates/grading/UploadImagesForm.twig:7]: Why is this text necessary anyway?  Would it make sense to just put the help icon next to the text below this?

Comment (musaabimran) [reply site/app/templates/grading/UploadImagesForm.twig:7]: I think this text should be there, sometimes users don't get focused on the help icon.

**Review by Youssef-Chip — CHANGES_REQUESTED** [review]
image
Can you make it like these help icons?

**Comment by musaabimran** [comment]
> image Can you make it like these help icons?

Sure! I'll update.

**Review by Youssef-Chip — APPROVED** [review]

**Pull request merged**